src/libs/ajax/data-table-providers/WdsDataTableProvider.ts

```diff
@@ -84,9 +84,9 @@ const getRelationParts = (val: unknown): string[] => {
 
 // Extract wds URL from Leo response. exported for testing
 export const getWdsUrl = apps => {
-  // look explicitly for an app named 'cbas-wds-default'. If found, use it, even if it isn't running
+  // look explicitly for an app named 'wds-${app.workspaceId}'. If found, use it, even if it isn't running
   // this handles the case where the user has explicitly shut down the app
-  const namedApp = apps.filter(app => app.appType === 'CROMWELL' && app.appName === 'cbas-wds-default')
+  const namedApp = apps.filter(app => app.appType === 'CROMWELL' && app.appName === `wds-${app.workspaceId}`)
```

Comment:
I find it confusing that the appType is `CROMWELL`, but the `appName` is wds (which I thought was a different app). Can you explain?

Reply:
`CROMWELL` is an enumeration required by Leo to invoke the correct Helm charts for deploying WDS, etc.: See here: https://leonardo.dsde-dev.broadinstitute.org/#/appsV2/createAppV2